content/bn/cloud-native-apps.md

```diff
@@ -0,0 +1,31 @@
+---
+title: ক্লাউড নেটিভ অ্যাপস (Cloud Native Apps)
+status: Completed
+category: ধারণা
+tags: ["অ্যাপ্লিকেশন", "", ""]
+---
+
+## এটা কি
+
+ক্লাউড নেটিভ অ্যাপ্লিকেশনগুলি বিশেষভাবে [ক্লাউড কম্পিউটিং](/cloud-computing/)-এ উদ্ভাবনের সুবিধা নেওয়ার জন্য ডিজাইন করা হয়েছে।
+এই অ্যাপ্লিকেশনগুলি তাদের নিজ নিজ ক্লাউড আর্কিটেকচারের সাথে সহজেই একত্রিত হয়, ক্লাউডের সংস্থান এবং [স্কেলিং](/scalability/) ক্ষমতার সুবিধা নিয়ে থাকে।
+এটি ক্লাউড কম্পিউটিং দ্বারা চালিত অবকাঠামোতে(infrastructure) উদ্ভাবনের সুবিধা গ্রহণকারী অ্যাপ্লিকেশনগুলিকেও বোঝায়।
+ক্লাউড নেটিভ অ্যাপ্লিকেশনগুলি আজকে এমন অ্যাপগুলিকে অন্তর্ভুক্ত করে যা একটি ক্লাউড প্রদানকারীর ডেটাসেন্টারে(data-center) এবং ক্লাউড নেটিভ প্ল্যাটফর্মে অন-প্রিমিসে(on-premise) চলে।
+
+## এটা যেসব সমস্যাতে দৃষ্টিপাত করে
+
+ঐতিহ্যগতভাবে, অন-প্রিমিস পরিবেশগুলি মোটামুটি পছন্দসই উপায়ে গণনা সংস্থান সরবরাহ করে।
+প্রতিটি ডেটাসেন্টারের(data-center) পরিষেবা ছিল যা নির্দিষ্ট পরিবেশে অ্যাপ্লিকেশনগুলিকে [শক্তভাবে সংযুক্ত](/tightly-coupled-architectures/) করে,
+প্রায়শই [ভার্চুয়াল মেশিন](/virtual-machine/) এবং পরিষেবার মতো অবকাঠামোর(infrastructure) জন্য ম্যানুয়াল প্রভিশনিংয়ের(manual provisioning) উপর অনেক বেশি নির্ভর করে।
+যার ফল স্বরূপ, ডেভেলপার এবং তাদের অ্যাপ্লিকেশনগুলিকে সেই নির্দিষ্ট ডেটা সেন্টারে সীমাবদ্ধ করে।
+ক্লাউডের জন্য ডিজাইন করা হয়নি এমন অ্যাপ্লিকেশনগুলি ক্লাউড পরিবেশের স্থিতিস্থাপকতা এবং স্কেলিং(scaling) ক্ষমতার সুবিধা নিতে পারে না।
+উদাহরণস্বরূপ, সঠিকভাবে শুরু করার জন্য ম্যানুয়াল হস্তক্ষেপের(manual intervention) প্রয়োজন এমন অ্যাপগুলি স্বয়ংক্রিয়ভাবে স্কেল(scale) করতে পারে না,
+অথবা ব্যর্থতার ক্ষেত্রে সেগুলি স্বয়ংক্রিয়ভাবে পুনরায় চালু করা যাবে না।
+
+## এটা কিভাবে সাহায্য করে
+
+যদিও ক্লাউড নেটিভ অ্যাপ্লিকেশনগুলির জন্য কোনও "এক মাপ সব ক্ষেত্রে মানানসই" পথ নেই, তবে তাদের কিছু মিল রয়েছে৷
+ক্লাউড নেটিভ অ্যাপগুলি স্থিতিস্থাপক, পরিচালনাযোগ্য এবং তাদের সাথে থাকা ক্লাউড পরিষেবাগুলির স্যুট(suite) দ্বারা সাহায্য করা হয়৷
+বিভিন্ন ক্লাউড পরিষেবাগুলি উচ্চ মাত্রার(high degree) [পর্যবেক্ষণযোগ্যতা](observability)(/observability/) সক্ষম করে,
+সমস্যাগুলি বৃদ্ধির আগে ব্যবহারকারীদের সনাক্ত করতে এবং সমাধান করতে সক্ষম করে।
+দৃঢ় অটোমেশনের(automation) সাথে মিলিত হয়ে তারা ইঞ্জিনিয়ারদের(engineer) ন্যূনতম পরিশ্রমের সাথে ঘন ঘন এবং অনুমানযোগ্যভাবে(predictably) উচ্চ-প্রভাবিত(high-impact) পরিবর্তন করতে দেয়।
```

Comment:
I don't find any meaning in this sentence in Bengali. It would be great if you could make a suitable sentence from English.

Reply:
Yes, seems good.